HomeWork_10/Main.java

```diff
@@ -0,0 +1,62 @@
+package com.company;
+
+import java.io.FileNotFoundException;
+import java.io.UnsupportedEncodingException;
+import java.util.ArrayList;
+
+/**
+ * Main class of the application. Contains the entry point.
+ */
+public class Main {
+
+    /**
+     * The entry point of application. Reads ip addresses from command line or file,
+     * simulates poll of servers and writes results to file.
+     * @param args command line launch parameters.
+     */
+    public static void main(String[] args) {
+        int maxPing = 0;
+        String[] ip;
+        if (args.length != 0) {
+            ip = args;
```

Comment:
There is a good place to show your OOP knowledge. Create two readers instead this conditional approach